Automattic/jetpack
# Forms: Allow fields style sync

## Proposed changes:

* Include an option in the Form fields Sidebar to allow styles synchronization between the fields

![image](https://user-images.githubusercontent.com/7811225/219178049-9b8ff099-be0e-4622-b3f7-aca43eb27945.png)

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Create a post and add a Form block
* By default, the style sync is enabled, so changing any field style should affect the other fields
* Play with the style sync option and change the fields style to check if it work as expected
